src/main/java/com/untamedears/jukealert/database/JukeAlertDAO.java

```diff
@@ -270,6 +272,9 @@ public void insert(Snitch snitch) {
 		} catch (SQLException e) {
 			logger.log(Level.SEVERE, "Failed to insert new snitch: ", e);
 		}
+		if (snitch.hasAppender(LeverToggleAppender.class)) {
+			setToggleLever(snitch.getId(), false);
+		}
```

Comment:
Because you have an overloaded persist() method in the LevelToggleAppender, I do not believe you need to or should have this code or its buddy in update(), as the code here will just have the effect of bypassing the ID = -1 protection you have in the persist() of LevelToggleAppender.